### DISCUSSION TITLE
Using Mixamo models

### DISCUSSION THREAD

**Discussion by sandru-nolimit** [jnsmalm/pixi3d#201] [General]
Hi everyone! I'm using two [Mixamo ](https://www.mixamo.com) fbx files converted to glb with [FBX2glTF](https://github.com/facebookincubator/FBX2glTF).
The first file called _characterModel.glb_ has the character with texture in a static T-pose position, let's call it Goblin.
The second file is called _animationsModel.glb_ and it contains an animation from Mixamo without skin/texture, a capoeira dance.

How can I play an animation from _animationsModel_ to the character? I want the Goblin to do capoeira.

PS: If I download an animation from Mixamo together with the skin it behaves well, no issues here. But I want to apply the same animation to multiple characters at once and I don't know how. If download each character with a custom animation I will end up with a huge project size. 
PPS:  i'm using Pixijs for about 7 years, but it's the first time i'm using **pixi3d** and I couldn't find a solution to my problem, not here or anywhere. I'm not an experimented 3D guy, but I guess it should be a way to translate an animation to a character that is using the same bones system.

**Comment by jnsmalm** [comment]
Hey! For skinned meshes, you try just copy the skin.

```
model2.meshes.forEach((mesh, i) => {
  mesh.skin = model.meshes[i].skin;
});
```

This works if the meshes matches on the models. Otherwise you have to create something more sophisticated :)

**Comment by sandru-nolimit** [comment]
Hi. Unfortunately, the solution didn't worked for me. I need to investigate a bit mode on the Mixamo exported models. And most probably, yes, I will have to create something more sofisticated. 

Anyway, thanks for the answer! I'll get back here with a solution once I'll figure it out.
